squad/core/migrations/0130_project_status_baseline_next.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.12 on 2020-05-14 10:20
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0129_projectstatus_nullable_notified_on_timeout'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='projectstatus',
+            name='baseline',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='next_statuses', to='core.Build'),
```

Comment:
For some reason here we get `related_name='next_statuses'` whereas in the model it's `related_name='next_status'`

Reply:
typo, thanks for spotting.

Reply:
Fixed. Also migration/model had field type mismatch